Extensions/TextObject/textruntimeobject-pixi-renderer.js

```diff
@@ -105,6 +105,14 @@ gdjs.TextRuntimeObjectPixiRenderer.prototype.getHeight = function() {
     return this._text.height;
 };
 
+gdjs.TextRuntimeObjectPixiRenderer.prototype.setWidth = function(newWidth) {
+    this._text.width = newWidth;
```

Comment:
Same, what is the result on screen of doing this?

Reply:
Let's not implement this - it's ok for objects to "not follow orders" and don't update their size.